test.sh

```diff
@@ -29,4 +29,5 @@ go vet $(go list ./... | grep -v vendor)
 
 remove_containers
 
+curl -X POST https://mermade.org.uk/api/v1/validate -H "Accept: application/json" -H "Content-Type: multipart/form-data" -F "source=`cat docs/openapi.yml`" | go run test/openapi-validate.go
```

Comment:
2 things: 

* can nuke the go file for python, grep or jq (jq likely not installed, first 2 / other preferably)
* can we use something that we control or that is offical to do this validation ? this is hosted by some dude on his site (granted, he looks reputable)

Reply:
There's no official validation tool yet released.

Reply:
is there any tool that we can run in a container instead of an http endpoint then? just don't want our builds to go down because some guy's personal server is offline for a day.

Reply:
According to official OpenAPI doc, there are couple libraries we can use to build a validation tool, but nothing we can use just out of the box.

Reply:
https://github.com/OAI/OpenAPI-Specification/blob/master/IMPLEMENTATIONS.md

Reply:
Node lib: https://github.com/mermade/swagger2openapi

Reply:
ruby lib looks simple: https://github.com/kevindew/openapi3_parser

Reply:
@rdallman I made validation tool: https://github.com/fnproject/dockers/pull/8. This PR was updated, until that moment when fnproject/openapiv3-validator:0.0.1 will reach docker hub CI will remain red.